content/desktop/release-notes.md

```diff
@@ -21,6 +21,54 @@ Docker Desktop versions older than 6 months from the latest release are not avai
 
 Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1) to see what's coming next.
 
+## 4.34.0
+
+{{< release-date date="2024-08-29" >}}
+
+{{< desktop-install-v2 all=true beta_win_arm=true version="4.34.0" build_path="/TBD/" >}}
+
+### New
+
+- [Host networking](/engine/network/drivers/host.md#docker-desktop) support on Docker Desktop is now generally available.
+- If you authenticate via the CLI without Docker Desktop installed, you can now authenticate through a browser-based flow, removing the need for manual PAT generation.
+- Windows now supports automatic reclamation of disk space in Docker Desktop for WSL2.
+- Windows registry key sign in enforcement method in GA.
+- Fresh installations of Docker Desktop now use the containerd image store by default.
+- Improved the output of `docker image list` to show multi-platform-related image information.
+
+### Upgrades
+
+- [Docker Engine v27.1.2](https://docs.docker.com/engine/release-notes/27.1/#2712)
+- [containerd v1.7.20](https://github.com/containerd/containerd/releases/tag/v1.7.20)
+- [Docker Scout CLI v1.13.0](https://github.com/docker/scout-cli/releases/tag/v1.13.0)
+- [Docker Buildx v0.16.2](https://github.com/docker/buildx/releases/tag/v0.16.2)
+- Linux kernel `v6.10.1`
+
+### Bug fixes and enhancements
+
+#### For all platforms
+
+- Fixed a bug that caused the CLI to become idle when a container was started with AutoRemove (`--rm`) but whose port bindings would be rejected by Docker Dekstop at start-up.
+- Fixed a bug where diagnostics collection would fail sporadically on the **Support** screen.
+- Fixed a bug where folders wouldn't expand in a container's **File** tab. Fixes [docker/for-win#14204](https://github.com/docker/for-win/issues/14204).
+- Compose File Viewer can now convert and deploy your compose project to a kubernetes cluster
+- In-app updates now respect the proxy settings.
+- Extended the ECI Docker socket mount permissions feature to optionally child images derived from allowed images. This allows ECI to work with buildpacks (e.g., Paketo) that create ephemeral local images that use Docker socket mounts.
```

Comment:
@ctalledo shall we include this entry, since the PR with the additional fix to this has not been included?

Reply:
Hi @MihaelaStoica , actually let's keep the release note, because DD 4.34 already fixes the problem for the customer that requested it. There's an improvement coming in DD 4.35, but it should not prevent us from exposing this in DD 4.34. I will update the Docker Desktop docs accordingly.